corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ereReader/EREMentionRelationReader.java

```diff
@@ -0,0 +1,173 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.nlp.corpusreaders.ereReader;
+
+import edu.illinois.cs.cogcomp.core.datastructures.Pair;
+import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.*;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.aceReader.SimpleXMLParser;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.aceReader.XMLException;
```

Comment:
We should try to move these XML parsers outside the ACE Reader namespace. Future refactoring maybe.